### ISSUE TITLE
关于损失函数loss_conf的疑问

### ISSUE THREAD

**Issue by ad50810344** [bubbliiiing/yolo3-pytorch#47]
loss_conf = torch.sum(BCELoss(conf, mask) / batch_size * mask) + torch.sum(BCELoss(conf, mask) / batch_size * noobj_mask)
这一步总觉得有问题，负采样的地方为什么还是BCELoss(conf, mask)呢？

**Comment by bubbliiiing** [comment]
一样的，mask在负样本的时候是0。后面乘的是noobj_mask
